README.md

```diff
@@ -76,7 +76,7 @@ This meta-role requires a permission policy similar to the following:
 |------|-------------|:----:|:-------:|:--------:|
 | account_ids | AWS account IDs that are allowed to assume the role. | list(string) | [] | no |
 | entity_name | The name of the entity that the role is being created for (e.g. "test-user" or "host.example.com"). | string | | yes |
-| iam_username | The username of the IAM user allowed to assume the role.  If not provided, defaults to allowing any user in the specified account(s). | string | `root` | no |
+| iam_usernames | The list of IAM usernames allowed to assume the role.  If not provided, defaults to allowing any user in the specified account(s).    Note that including "root" in this list will override any other usernames in the list.| string | `["root"]` | no |
```

Comment:
You need to update the variable type here as well.

Reply:
Good catch, thanks!  Corrected in https://github.com/cisagov/ssm-read-role-tf-module/pull/2/commits/a078c8548005d7df2100791397cf59b57265c640.